app/views/ShareListView/index.js

```diff
@@ -138,43 +137,15 @@ class ShareListView extends React.Component {
 		const { searching } = this.state;
 		const { navigation, theme } = this.props;
 
-		if (isIOS) {
-			navigation.setOptions({
-				header: () => (
-					<ShareListHeader
-						searching={searching}
-						initSearch={this.initSearch}
-						cancelSearch={this.cancelSearch}
-						search={this.search}
-						theme={theme}
-					/>
-				)
-			});
-			return;
-		}
-
 		navigation.setOptions({
-			headerLeft: () => (searching
-				? (
-					<HeaderButton.Container left>
-						<HeaderButton.Item title='cancel' iconName='close' onPress={this.cancelSearch} />
-					</HeaderButton.Container>
-				)
-				: (
-					<HeaderButton.CancelModal
-						onPress={ShareExtension.close}
-						testID='share-extension-close'
-					/>
-				)),
-			headerTitle: () => <ShareListHeader searching={searching} search={this.search} theme={theme} />,
-			headerRight: () => (
-				searching
-					? null
-					: (
-						<HeaderButton.Container>
-							<HeaderButton.Item iconName='search' onPress={this.initSearch} />
-						</HeaderButton.Container>
-					)
+			header: () => (
+				<ShareListHeader
```

Comment:
You've unified the header behaviour here, but it's different from RoomsListView's header.
Can we use a closer UI from RoomsListView?